Makefile

```diff
@@ -38,7 +38,7 @@ wallet-web:
 	@scripts/build_wallet_web.sh
 
 .PHONY: wallet-web-docker
-wallet-web-docker: clean wallet-web
+wallet-web-docker: wallet-web
```

Comment:
is this intentional? to remove clean here?

Reply:
yes